websites/R/Rust Lang Book/presence.ts

```diff
@@ -0,0 +1,41 @@
+const presence = new Presence({
+	clientId: "1033608073106968576",
+});
+
+presence.on("UpdateData", async () => {
+	const path = window.location.pathname
+			.replace("/book/", "")
+			.replace(".html", ""),
+		presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/kTnc40Q.png", // rust logo 512x512
+			smallImageKey: "fhttps://i.imgur.com/B7Rip9I.png", // ferris 512x512
+			smallImageText: "Hi rustaceans!",
+			buttons: [
+				{
+					label: "Read the book",
```

Comment:
Shorter buttons are preferable
```suggestion
					label: "Read Book",
```